<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtTtTestModelGenerationDialog</class>
 <widget class="QDialog" name="mdtTtTestModelGenerationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test generation</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbTestModelType">
     <property name="text">
      <string>Test type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbTestModelType"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbTestNode">
     <property name="text">
      <string>Test node:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbTestNode"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbTestCable">
     <property name="text">
      <string>Test cable:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbTestCable"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbMeasureConnectionA">
     <property name="text">
      <string>Measure connection A:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbMeasureConnectionA"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbMeasureConnectionB">
     <property name="text">
      <string>Measure connection B:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbMeasureConnectionB"/>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mdtTtTestModelGenerationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mdtTtTestModelGenerationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
